triage/roster_log_review_queue/provenance.py

```
@@ -0,0 +1,63 @@
+    ZoneInfo = None  # type: ignore
+
+from .models import LiveCFPatchStats
+
+
+def _now_iso(tz_name: str = "America/New_York") -> str:
+    if ZoneInfo is not None:
+        try:
+            tz = ZoneInfo(tz_name)
+            return datetime.now(tz).isoformat()
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)